### PR TITLE
Update service-accounts-admin.md

### PR DESCRIPTION
**Pull request by islinwb**
update a broken link and fix typos

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4872)

### CODE REVIEW

File: docs/admin/service-accounts-admin.md [hunk 1]
```diff
@@ -8,14 +8,14 @@ title: Managing Service Accounts
...
-the [User Guide to Service Accounts](/docs/user-guide/service-accounts).*
+the [User Guide to Service Accounts](/docs/tasks/configure-pod-container/configure-service-account/).*
 
 *Support for authorization and user accounts is planned but incomplete.  Sometimes
 incomplete features are referred to in order to better describe service accounts.*
 
 ## User accounts vs service accounts
 
-Kubernetes distinguished between the concept of a user account and a service accounts
+Kubernetes distinguished between the concept of a user account and a service account
```
Comment (chenopis) [inline docs/admin/service-accounts-admin.md]: suggested edit: "Kubernetes `distinguishes` between the concept of a user account and a service account for a number of reasons:"

**Comment by islinwb** [comment]
@chenopis Thanks for reviewing. Change has been made as requested.

**Comment by chenopis** [comment]
@islinwb When you make a change, just give me a few days to respond. I didn't get a chance to look at this over the weekend. It looks okay now, so I don't know why you closed the PR.

**Pull request closed without merging**